tests/integration.rs

```diff
@@ -5921,6 +5921,147 @@ mod tests {
 
             handle_child_output(r, &output);
         }
+
```

Comment:
Typo: you have two `Signed-off-by: Bo Chen <chen.bo@intel.com>` in your commit `tests: Add integration test for vfio-user with SPDK NVMe`

Reply:
Good catch. Fixed!